janusgraph-solr/src/test/resources/solr/core-template/schema.xml

```diff
@@ -547,6 +547,7 @@
                prefixTree="geohash"/>
```

Comment:
I can, but should it also be updated in the imports for JtsGeoshapeHelper.java and JtsGeoshapeHelperTest.java?